### PR TITLE
Xiaomi Smart Home Binding - code review followup + battery level report

### PR DESCRIPTION
**Pull request by kubawolanin**
Signed-off-by: Kuba Wolanin <hi@kubawolanin.com>

### CODE REVIEW

**Comment by kubawolanin** [comment]
@pboos pinging you for awareness! :)

File: addons/binding/org.openhab.binding.mihome/.classpath [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kubawolanin) [inline addons/binding/org.openhab.binding.mihome/.classpath:1]: Turns out that `.classpath` and `.project` files are needed.

File: addons/binding/org.openhab.binding.mihome/ESH-INF/thing/bridge.xml [hunk 2]
```diff
@@ -16,22 +18,25 @@
         <config-description>
```
Comment (kubawolanin) [inline addons/binding/org.openhab.binding.mihome/ESH-INF/thing/bridge.xml:18]: @pboos I'd still opt for merging togeter `bridge` and `gateway`. 
Take a look e.g. at [Z-Wave Serial Controller](https://github.com/openhab/org.openhab.binding.zwave/blob/3133e344fea4299720f06d0bf26181bdb177496c/ESH-INF/thing/_controller_serial.xml) - it's a matter of passing `channel-type` configurations to the bridge.xml file.

I'm not sure how to refactor `XiaomiBridgeHandler` to match the logic though.

File: addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiItemHandler.java [hunk 3]
```diff
@@ -256,6 +250,15 @@ public void onItemUpdate(String sid, String command, JsonObject message) {
                     }
                     break;
             }
+
+            if (data.get("voltage") != null) {
+                Integer voltage = data.get("voltage").getAsInt();
+                updateState(CHANNEL_VOLTAGE, new DecimalType(voltage));
+
+                if (voltage < 2800) {
+                    triggerChannel("batteryLevel", "LOW");
```
Comment (kubawolanin) [inline addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiItemHandler.java:259]: @pboos not sure if I'm doing it properly ;)

**Comment by pboos** [comment]
Looks good. Thank you!! :)
Not sure if everything is 100% correct (the voltage and low battery thing), but still better than not being in there at all :). Testing kind of hard since need to get the battery low, which might take a while :D.

About merging the gateway (bridge + thing) into one. Will have to look into it. But currently not much time. But if possible, that definitely is the better way!

**Comment by hamwong** [comment]
hi, I try to look around in forum and can't found solution,
can you advise where can I download the newest snapshot in jar?

**Pull request merged**